tock/employees/views.py

```diff
@@ -1,17 +1,22 @@
 import datetime
+import json
 
 from django.contrib.auth.models import User
+from django.db.models import Sum
 from django.core.urlresolvers import reverse
 from django.views.generic import ListView, DetailView
 from django.views.generic.edit import FormView
 from django.shortcuts import get_object_or_404
 
 
 from tock.utils import PermissionMixin, IsSuperUserOrSelf
+from .utils import get_last_n_rp, get_rps_in_fy, calculate_utilization, utilization_by_range, get_fy_first_day
 
 from .forms import UserForm
 from .models import UserData
 
+from hours.models import ReportingPeriod, TimecardObject
```

Comment:
We're not actually using ReportingPeriod in this file, so we should be able to nix the import.